redux-todomvc/store/configureStore.js

```diff
@@ -1,9 +1,14 @@
 import { compose, createStore, applyMiddleware } from 'redux'
 import rootReducer from '../reducers'
 
+import {marionetteMiddleware, marionetteDispatch} from '../../marionette-redux.js';
+
 export default function configureStore(initialState) {
   const store = compose(
+    applyMiddleware(marionetteMiddleware(window.Backbone, window.Backbone.Marionette, window._))
   )(createStore)(rootReducer, initialState)
 
+  marionetteDispatch(store.dispatch, window.Backbone, window._);
+
```

Comment:
You have to pass in Backbone, Marionette, and underscore because your app old app may use globals, requirejs, webpack, etc. and we're not going to judge.